invenio_drafts_resources/resources/drafts/resource.py

```diff
@@ -138,10 +138,16 @@ def create(self):
         except AttributeError:
             raise ActionNotImplementedError(cmd_name)
 
-        # NOTE: Due to the route we assume the commands only need
-        # id_ and identity
-        item = cmd_func(
-            resource_requestctx.route["pid_value"],
-            g.identity
-        )
+        if cmd_name == "publish":
+            item = cmd_func(
+                resource_requestctx.route["pid_value"],
+                g.identity,
+                links_config=self.config.record_links_config
```

Comment:
Question: Wouldn't this be potentially needed by all?

Reply:
A priori this is the `DraftActionResource`, so I would assume that most would return a `draft_links_config` over `record_links_config (`"publish"` action is special). But we only have 1 case, so let's wait and see...